ci/pipelines/e2e-helpers.yml

```diff
@@ -614,40 +660,49 @@ jobs:
       put: slack-notification
       attempts: 10
       params:
-        channel: '#govuk-pay-starling'
+        channel: "#govuk-pay-starling"
         silent: true
-        text: ':red-circle: Failed to build and push pay-stubs candidate image - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>'
+        text: ":red-circle: Failed to build and push pay-stubs candidate image - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>"
         icon_emoji: ":concourse:"
         username: pay-concourse
     on_success:
       put: slack-notification
       attempts: 10
       params:
-        channel: '#govuk-pay-activity'
+        channel: "#govuk-pay-activity"
         silent: true
-        text: ':green-circle: Built and pushed pay-stubs candidate image - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>'
+        text: ":green-circle: Built and pushed pay-stubs candidate image - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>"
         icon_emoji: ":concourse:"
         username: pay-concourse
 
   - name: stubs-e2e
     plan:
       - in_parallel:
-        - get: stubs-candidate-ecr-registry-test 
-          passed: [build-and-push-stubs-candidate]
-          trigger: true
-          params:
-            format: oci
-        - get: pay-ci
+          - get: stubs-candidate-ecr-registry-test
+            passed: [build-and-push-stubs-candidate]
+            trigger: true
+            params:
+              format: oci
+          - get: pay-ci
+          - task: parse-candidate-tag
+            file: pay-ci/ci/tasks/parse-candidate-tag.yml
+            input_mapping:
+              ecr-repo: stubs-candidate-ecr-registry-test
+          - load_var: release_number_tag
+            file: tags/release-number
```

Comment:
As per the comments above - these `task: parse-candidate-tag` and `load_var` steps will need to be de-indented